mock-core-configs/etc/mock/templates/amazonlinux-2023.tpl

```diff
@@ -0,0 +1,53 @@
+config_opts['root'] = 'amazonlinux-2023-{{ target_arch }}'
+config_opts['chroot_setup_cmd'] = 'install --allowerasing @buildsys-build dnf'
+config_opts['dist'] = 'amzn2023' # only useful for --resultdir variable subst
+config_opts['plugin_conf']['ccache_enable'] = False
+config_opts['package_manager'] = 'dnf'
+config_opts['description'] = 'Amazon Linux 2023'
+config_opts['releasever'] = 'latest'
+
+config_opts['bootstrap_image'] = 'public.ecr.aws/amazonlinux/amazonlinux:2023'
+
+config_opts['dnf.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+sysleg_device=
+install_weak_deps=0
+user_agent={{ user_agent }}
+
+[amazonlinux]
+name=Amazon Linux 2023 repository
```

Comment:
Could you add `- $basearch` to end of the string here, so that it's obvious what arch is being used in mock logs?

Reply:
```suggestion
name=Amazon Linux 2023 repository - $basearch
```

Reply:
Done